src/tests/system/tests/test_memcache.py

```diff
@@ -1,1660 +1,541 @@
+    :title: Validate memory cache behavior for users and groups with all caches disabled
     :setup:
-        1. Add 'group1', 'group2' and 'group3' to SSSD
-        2. In SSSD nss change 'memcache_size_passwd' to '0'
+        1. Create user `user1` and group `group1`.
+        2. Disable all caches in SSSD configuration.
         3. Start SSSD
     :steps:
-        1. Find 'group1', 'group2' and 'group3' with getent.group(name)
-        2. Check that groups have correct names
-        3. Stop SSSD
-        4. Find 'group1', 'group2' and 'group3' with getent.group(name)
-        5. Check that groups have correct names
+        1. Perform lookup operation for `user1` and `group1`.
+        2. Stop SSSD and perform lookup again for `user1` and `group1`.
     :expectedresults:
-        1. Groups are found
-        2. Groups have correct names
-        3. SSSD is stopped
-        4. Groups are found
-        5. Groups have correct names
+        1. `user1` and `group1` are found.
+        2. `user1` and `group1` are NOT found.
     :customerscenario: False
     """
 
-    def check(groups):
-        for group in groups:
-            result = client.tools.getent.group(group)
-            assert result is not None, f"Group {group} was not found using getent"
-            assert result.name == group, f"Groupname {result.name} is incorrect, {group} expected"
-
-    groups = ["group1", "group2", "group3"]
-    for group in groups:
-        provider.group(group).add()
+    provider.user("user1").add()
+    provider.group("group1").add()
 
     client.sssd.nss["memcache_size_passwd"] = "0"
-    client.sssd.start()
-
-    check(groups)
-    client.sssd.stop()
-    check(groups)
-
-
-@pytest.mark.importance("high")
-@pytest.mark.cache
-@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
-def test_memcache__user_cache_is_disabled_and_lookup_users(client: Client, provider: GenericProvider):
-    """
-    :title: Lookup user by name when SSSD is stopped and 'memcache_size_passwd' = 0
-            uses memory cache therefore user is not found
-    :setup:
-        1. Add users to SSSD
-        2. Set users uids and gids
-        3. In SSSD nss change 'memcache_size_passwd' to '0'
-        4. Start SSSD
-    :steps:
-        1. Find 'user1', 'user2' and 'user3' with id(name)
-        2. Check that users have correct names
-        3. Stop SSSD
-        4. Find users with id(name)
-        5. Find users with id(uid)
-    :expectedresults:
-        1. Users are found
-        2. Users have correct names
-        3. SSSD is stopped
-        4. Users are not found
-        5. Users are not found
-    :customerscenario: False
-    """
-    ids = [("user1", 10001), ("user2", 10002), ("user3", 10003)]
-    for user, id in ids:
-        provider.user(user).add(uid=id, gid=id + 500)
+    client.sssd.nss["memcache_size_group"] = "0"
+    client.sssd.nss["memcache_size_initgroups"] = "0"
```

Comment:
Not needed to disable `memcache_size_initgroups` since it's not used in this test.